.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,57 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    types: [ opened, synchronize, reopened ]
+    branches: [ master, "bugfix/*", "feature/*", "release/*" ]
+  schedule:
+    - cron: '34 2 * * 0'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  analyze:
+    name: Code Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'java' ]
```

Comment:
Does this not need Kotlin on the list ?

Reply:
Not for now, if we look at [here](https://codeql.github.com/docs/codeql-language-guides/codeql-for-java/) it states:

> CodeQL treats Java and Kotlin as parts of the same language, so to enable Kotlin support you should **enable java as a language**.